MobileWallet/TariLib/Wrappers/WalletTestDataExtension.swift

```diff
@@ -41,40 +41,52 @@
 import Foundation
 
 extension Wallet {
-    func generateTestData() throws {
-        let didGenerateData = wallet_test_generate_data(self.pointer)
+    func generateTestData(filePath: String) throws {
```

Comment:
Don't think this should accept a path, instead use the wallet constant. Else test data could be added to a wallet that was never created

Reply:
Fair point